annotations/raml2java.raml

```diff
@@ -0,0 +1,100 @@
+#%RAML 1.0 Library
+uses:
+  commons: "https://raw.githubusercontent.com/apiregistry/commons/master/commons.raml"
```

Comment:
Should we include the commons RAML in here as well, or what is that exactly. I would argue to minimize the amount of external dependencies.